toxcore/Messenger.c

```diff
@@ -2243,6 +2329,28 @@ static int m_handle_packet(void *object, int i, const uint8_t *temp, uint16_t le
             break;
         }
 
+        case PACKET_ID_INVITE_GROUPCHAT: {
+#ifndef VANILLA_NACL
+
```

Comment:
can we write the next few lines a bit better?
lots of "magic" numbers are repeated with copy and paste
```2 + GC_JOIN_DATA_LENGTH```
```data + 2```
```data_length - 2```